logsapi/server.go

```diff
@@ -64,14 +64,22 @@ func handler(libhoneyClient *libhoney.Client) http.HandlerFunc {
 			switch v := msg.Record.(type) {
 			case string:
 				// attempt to parse as json
-				var record map[string]interface{}
-				err := json.Unmarshal([]byte(v), &record)
+				var jsonRecord map[string]interface{}
+				err := json.Unmarshal([]byte(v), &jsonRecord)
 				if err != nil {
+					// not JSON; we'll treat this log entry as a timestamped string
 					event.Timestamp = parseMessageTimestamp(event, msg)
 					event.AddField("record", msg.Record)
 				} else {
-					event.Timestamp = parseFunctionTimestamp(msg, record)
-					event.Add(record)
+					// we've got JSON
```

Comment:
you've got JSON

Reply:
![You've got mail!](https://user-images.githubusercontent.com/517302/118271085-d792a600-b48e-11eb-936b-5a191078eefb.gif)